### PR TITLE
MenuCommand class

### PR DESCRIPTION
**Pull request by thaddyyz**
left original menu class for reference, remove it if not needed. Additionally can identify a suitable place to call GetMenu as it should only be called once when the program is running.

### CODE REVIEW

File: src/main/java/seedu/duke/MenuCommand.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+     * MenuCommand calls access to file to retrieve stored menu and 
+     * prints menu in the format of index, foodname and price.
+     * */
+    public MenuCommand() throws FileNotFoundException { 
+        try {
+            getMenu();
+        } catch (FileNotFoundException e) {
+            throw new FileNotFoundException("Menu file not found!");
+        }
+        Ui.printMenuHeader();
```
Comment (andrewtkh1) [inline src/main/java/seedu/duke/MenuCommand.java:23]: I believe the printing of header & the body should be in a `execute()` function.
The other comment below shows what I mean, or you can refer to the link below to see the deleteCommand class.

E.g) https://github.com/AY2122S1-CS2113-T13-2/tp/blob/master/src/main/java/seedu/duke/DeleteCommand.java

File: src/main/java/seedu/duke/MenuCommand.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        for (int i = 0; i < foodList.size(); i++) {
+            Ui.printMenu(i + 1, foodList.get(i), priceList.get(i));
+        }
+    }
+
```
Comment (andrewtkh1) [inline src/main/java/seedu/duke/MenuCommand.java:28]:
```suggestion
@Override
public void execute() {
    Ui.printMenuHeader();
    for (int i = 0; i < foodList.size(); i++) {
            Ui.printMenu(i + 1, foodList.get(i), priceList.get(i));
    }
}
```

File: src/main/java/seedu/duke/Ui.java [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 public class Ui {
 
+    static String line = "--------------------------------------------------";
```
Comment (andrewtkh1) [inline src/main/java/seedu/duke/Ui.java:7]: Perhaps can consider changing this to private static final?

File: src/main/java/seedu/duke/MenuCommand.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    /**
+     * Get menu data from foodData.txt
+     * */
+    private void getMenu() throws FileNotFoundException {
+        File myObject = new File("data/foodData.txt");
+        Scanner myReader = new Scanner(myObject);
+        while (myReader.hasNextLine()) {
+            String data = myReader.nextLine();
+            foodList.add(data.substring(0,data.indexOf("|")));
+            priceList.add(Double.parseDouble(data.substring(data.indexOf("|") + 1)));
```
Comment (andrewtkh1) [inline src/main/java/seedu/duke/MenuCommand.java:38]: Would it be better to not deeply nest the code? Might help to make it more readable.

**Review by andrewtkh1 — CHANGES_REQUESTED** [review]
View comments in code.

File: src/main/java/seedu/duke/MenuCommand.java (hunk 4 above)
Comment (thaddyyz) [reply src/main/java/seedu/duke/MenuCommand.java:55]: I still need this as a while loop but I put the entire loop in a separate function to improve readability

**Pull request merged**